app/assets/javascripts/components/story/CollapsedStory/CollapsedStoryIcon.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { iconRule, classIconRule } from 'rules/story';
+
+const CollapsedStoryIcon = ({ storyType }) => (
+  <span className='Story__icon'>
+    <i className={`mi md-${classIconRule(storyType)} md-16`}>{iconRule(storyType)}</i>
```

Comment:
Can we move these two to this same file as well? Also, since this file can be generic we could:

- rename it to just `StoryIcon`
- add a modifier in the `CollapsedStory` component like `Story--collapsed`
- change this style https://github.com/Codeminer42/cm42-central/blob/refactor/story-item/app/assets/stylesheets/new_board/_story.scss#L58 to be inside `&--collapsed` since these external rules (margin, display, ...) apply only when inside a collapsed story (this is gonna imply in changing from `&__icon` to `.Story__icon` as well, otherwise SCSS will generate `.Story--collapsed__icon` for us and that's not what we want)